src/Components/CreateImageWizard/steps/Oscap/index.tsx

```diff
@@ -101,77 +100,56 @@ const OscapContent = () => {
+        use of tailored policies, whereas OpenSCAP gives you the default
+        versions. This will automatically help monitor the adherence of your
+        registered RHEL systems to a selected policy or profile.
+      </Text>
       {complianceEnabled && (
-        <FormGroup>
-          <Radio
-            id="openscap radio openscap type"
-            label="OpenSCAP"
-            name="oscap-radio-openscap"
-            isChecked={complianceType === 'openscap'}
-            onChange={() => handleTypeChange('openscap')}
-          />
-          <Radio
-            id="openscap radio compliance type"
-            label="Insights compliance"
-            name="oscap-radio-compliance"
-            isChecked={complianceType === 'compliance'}
+        <ToggleGroup aria-label="Default with single selectable">
+          <ToggleGroupItem
+            text="Compliance policies"
+            buttonId="toggle-group-compliance"
+            isSelected={complianceType === 'compliance'}
             onChange={() => handleTypeChange('compliance')}
           />
-        </FormGroup>
-      )}
-      {(!complianceEnabled || complianceType === 'openscap') && (
-        <Text>
-          OpenSCAP enables you to automatically monitor the adherence of your
-          registered RHEL systems to a selected regulatory compliance profile.
-          <br />
-          <Button
-            component="a"
-            target="_blank"
-            variant="link"
-            icon={<ExternalLinkAltIcon />}
-            iconPosition="right"
-            isInline
-            href={COMPLIANCE_AND_VULN_SCANNING_URL}
-          >
-            Documentation
-          </Button>
-        </Text>
+          <ToggleGroupItem
+            text="OpenSCAP profiles"
+            buttonId="toggle-group-openscap"
+            isSelected={complianceType === 'openscap'}
+            onChange={() => handleTypeChange('openscap')}
+          />
+        </ToggleGroup>
       )}
-      {complianceType === 'compliance' && (
-        <Text>
-          Insights compliance enables you to monitor the adherence of your
-          registered RHEL systems to a selected compliance policy.
-          <br />
-          <Button
-            component="a"
-            target="_blank"
-            variant="link"
-            icon={<ExternalLinkAltIcon />}
-            iconPosition="right"
-            isInline
-            href={COMPLIANCE_URL}
-          >
-            Define new policies in Insights Compliance
-          </Button>
-          <br />
-          <Button
-            component="a"
-            target="_blank"
-            variant="link"
-            icon={<ExternalLinkAltIcon />}
-            iconPosition="right"
-            isInline
-            href={COMPLIANCE_AND_VULN_SCANNING_URL}
-          >
-            Documentation
-          </Button>
-        </Text>
+      {policies?.data?.length === 0 && complianceType === 'compliance' && (
```

Comment:
Just a thought, but the alert will not be shown in case policies or policies.data is undefined. This will probably happen only in case something is incorrect, and I think we want this, but I just wanted to mention it, in case it's not the intention.